### DISCUSSION TITLE
prisma migrate dev

### DISCUSSION THREAD

**Discussion by jonasgroendahl** [prisma/prisma#10450] [Q&A] [answered]
I accidently ran `prisma migrate dev` instead of `prisma migrate deploy` on my production database. Does that have any implications? Can I revert it?

**Comment by pantharshit00** [comment, marked as answer]
If there were no data loss warnings, there should be no implication. `migrate dev` just makes a new migration if the schema is out of sync with the database. You should be fine here.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
